modules/cockroachdb/cockroachdb_test.go

```diff
@@ -28,6 +28,7 @@ func TestCockroach_NotRoot(t *testing.T) {
 		url: "postgres://test@localhost:xxxxx/defaultdb?sslmode=disable",
 		opts: []testcontainers.ContainerCustomizer{
 			cockroachdb.WithUser("test"),
+      cockroachdb.WithStatements(),
```

Comment:
question: this looks like an anti pattern, could you clarify why this is desired? Same below.

Reply:
This is what i meant by my last comment. The `test` user that is being used here doesn't have `MODIFYCLUSTERSETTING`, so it cannot run the default statements as that privilege is needed for those. The fact that we have to pass this `cockroachdb.WithStatements()` option to disable the default is indeed weird (I asked about that in one of other convos) but I suppose the other alternative would be to not have these defaults be actual defaults and let the user pass those in.

Reply:
What's needed to enable this to work by default?

Reply:
you mean just for these tests? we'd need the user to have `MODIFYCLUSTERSETTING` privilege. But the problem will likely exist for users that try and set up their cluster with a non-root user anyways. Alternatively we make the defaults not defaults and just let the user add those statements if they want.

Reply:
Sorry, I don't think I fully understood the issue you described.

To confirm my understanding we need to disable the cluster settings here as we're using a custom user which doesn't have the required permissions?

If that's the case I would suggest just correcting the indentation and adding a comment to the line to make that clear.

Reply:
> To confirm my understanding we need to disable the cluster settings here as we're using a custom user which doesn't have the required permissions?
correct!

I'll re-format the code and add that comment then